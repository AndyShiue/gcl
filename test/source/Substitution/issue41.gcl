con F : Int -> Bool
var i, n : Int

{: P n = ⟨ ∧ i : 0 ≤ i < n : F i ⟩ 
:}

n := n + i
{ P n }