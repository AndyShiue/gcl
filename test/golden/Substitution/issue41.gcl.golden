P (n + i)
  P (n + i) ===> ⟨ ∀ ?i_1 : 0 ≤ ?i_1 < n + i : f ?i_1 ⟩
    